### DISCUSSION TITLE
No keyboard shortcuts for screenshots

### DISCUSSION THREAD

**Discussion by t3st-account** [regolith-linux/regolith-desktop#733] [Q&A] [answered]
Hello, I am running regolith 2.1 on top of Linux mint 21 (I know no ofifcial support)
and since the update to mint 21 (from 20.3 iirc) I have no Screenshot keybinds.
They are missing from the help dialogue ( mod+shif+?) and from control center.
does Anyone know how to reenable those? Either in config files or via GUI, I dont care...
But I especially need Selection screenshots.

[regolith-diag.txt](https://github.com/regolith-linux/regolith-desktop/files/9690427/regolith-diag.txt)

thank you all in advance

**Comment by fabcontigiani** [comment, marked as answer]
You can add your own screenshot keybindings via GNOME keybindings, managed in the Regolith Settings app, see [here](https://regolith-desktop.com/docs/using-regolith/configuration/#keybindings).

I recently accomplished the same by customizing one of Regolith's default partials located at `/usr/share/regolith/i3/config.d/90_user-programs` following [these](https://regolith-desktop.com/docs/howtos/customize-i3-configuration/#how-to-customize-a-default-partial) instructions.

Even if the file name suggests it's only for user-chosen startup programs, I decided to use it for this purpose too, and ended up adding the following lines:
```
# Run flameshot at startup
exec --no-startup-id flameshot
 
# Screenshot keybindings
bindsym --release Print exec flameshot gui
bindsym --release Shift+Print exec flameshot gui -c -s
```

In this case I used [flameshot](https://flameshot.org/), but feel free to use gnome-screenshot or any other screenshot utility you prefer.
